instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -317,6 +320,21 @@ private void applyConfiguration() {
                         return tracerProviderBuilder.addSpanProcessor(screenAttributesAppender);
                     });
         }
+
+        // Wire up the logging exporter, if enabled.
```

Comment:
My understanding about the use of `LoggingSpanExporter` was as a temporary replacement for Zipkin's until we implemented the OTLP exporters. So I'm a bit confused by these changes because we're now essentially adding the logger exporter as a first-class citizen feature but I'm not sure that the demand for this kind of feature is high enough to provide official support for it, or to justify the [extra dependency](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging) needed to make it work. Especially because it seems like the logs are printed using JUL, which doesn't seem too useful for Android in general, and also because I think it's easy to add it (or any other exporter that logs the signals) if needed by using the existing customizer apis.

Reply:
It's been my experience that developers want to see their generated app telemetry well in advance of actually sending it somewhere meaningful, and that's where the logging exporter comes into play.

You can definitely see the JUL logs with `logcat` when running in the emulator. I think you might also be able to see them with adb?

In any case, that's fine if you think this is premature. We can wait until people ask for it.

Reply:
I think the idea that people might want to see what they're sending in logs is reasonable, what I'm not sure about is if it needs to be an officially supported feature here, especially with how easy it is to add that functionality by using the existing customizers and the flexibility that it can provide when it comes to choosing how to print those logs too, as opposed to enforcing JUL an also an extra dependency, for a feature that seems mostly useful for development purposes.